.github/workflows/run_tests.yml

```diff
@@ -60,14 +60,14 @@ jobs:
 
     env:
       BUNDLE_GEMFILE: gemfiles/${{ matrix.gemfile }}.gemfile
-      APPOPTICS_GEM_TEST: true
-      APPOPTICS_REPORTER: file
-      APPOPTICS_COLLECTOR: /tmp/appoptics_traces.bson
-      APPOPTICS_MONGO_SERVER: "mongo"
-      APPOPTICS_RABBITMQ_SERVER: "rabbitmq"
-      APPOPTICS_MEMCACHED_SERVER: "memcached"
-      APPOPTICS_REPORTER_FILE_SINGLE: false
-      APPOPTICS_FROM_S3: true
+      SW_APM_GEM_TEST: true
+      SW_APM_REPORTER: file
+      SW_APM_COLLECTOR: /tmp/sw_apm_traces.bson
+      SW_APM_REPORTER_FILE_SINGLE: false
+      OBOE_FROM_S3: true
+      MONGO_SERVER: "mongo"
```

Comment:
Is it expected that these 3 lines don't need a `SW_APM` prefix?

Reply:
yes, removed for vars that are for test servers only